internal/logx/logx.go

```diff
@@ -0,0 +1,123 @@
+package logx
```

Comment:
I'm not convinced the log package needs to be renamed. We're never using standard library log except for in this file.

There are two other packages named `logx`, so this may be confusing just as well. I think we should just review properly that logging is always done through the `internal/log` package.
https://github.com/akaspin/logx
https://github.com/souhup/logx

Reply:
it doesn't. when I first started debugging in hover however it was extremely annoying to try and debug. because I kept typing log.Println("blah") and because this package shadows the stdlib package it never worked. =)

personal preference here would be a internal/logx package with child packages {error,warn,info,debug,trace} packages.

but whatever you want long as we get the verbosity in so hover is actually debuggable.